specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -593,20 +985,52 @@
     "HazardIndex": {
       "type": "integer",
       "format": "int32",
-      "description": "A severity/hazard index.\n  * `0` - No hazard.\n  * `1` - Be informed, be aware.\n  * `2` - Pay attention, be prepared.\n  * `3` - Take action.\n  * `4` - Life threatening, emergency."
+      "description": "A severity/hazard index.\n  * `0` - No hazard.\n  * `1` - Be informed, be aware.\n  * `2` - Pay attention, be prepared.\n  * `3` - Take action.\n  * `4` - Life threatening, emergency.",
+      "enum": [
+        0,
+        1,
+        2,
+        3,
+        4
+      ],
+      "x-ms-enum": {
+        "name": "HazardIndex",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": 0,
+            "description": "No hazard."
+          },
+          {
+            "value": 1,
+            "description": "Be informed, be aware."
+          },
+          {
+            "value": 2,
+            "description": "Pay attention, be prepared."
+          },
+          {
+            "value": 3,
+            "description": "Take action."
+          },
+          {
+            "value": 4,
+            "description": "Life threatening, emergency."
+          }
+        ]
```

Comment:
This most likely need short names to map to enum labels. Example: `EMERGENCY = 4`